bay/containers/container.py

```diff
@@ -220,7 +220,11 @@ def _parse_volume_options(self, options):
         if git_match:
             options["source"] = "../{}/{}".format(git_match.group(1), git_match.group(2).lstrip("/"))
         if "/" in options["source"]:
-            options["source"] = os.path.abspath(os.path.join(self.graph.path, options["source"]))
+            # Allow the volume mount root to be different if needed (e.g. Windows)
+            if os.environ.get("BAY_VOLUME_HOME"):
```

Comment:
can we add some minimal documentation about this new env variable?
may be `docs/commands.rst` would be a good place.